docs/framework/additional-apis/toc.md

```diff
@@ -1,4 +1,22 @@
 # [Additional class libraries and APIs](index.md)
+## Microsoft.SqlServer.Server
+### SmiOrderProperty Class
```

Comment:
Should there be separate pages for each type (SmiOrderPrpoerty, SqlStreamChars), and SqlChars link to the managed reference type?

Reply:
Not sure. I'm adding the ones that were failing apiscan.